exec/jdbc/src/main/java/org/apache/drill/jdbc/DrillPreparedStatement.java

```diff
@@ -32,4 +33,25 @@
  */
 public interface DrillPreparedStatement extends PreparedStatement {
 
+  /**
+   * @throws  SQLException
+   *            Any SQL exception
+   */
+  @Override
+  int getMaxRows() throws SQLException;
```

Comment:
What was the reason for declaring this method and the method below?They both are already available in `Statement` interface.

Reply:
I think I was declaring it because it seemed Avatica preferred using `setLargeMaxRows() / getLargeMaxRows()` . I, anyway, cannot really override this (all calls to `setMaxRows() / getMaxRows()` are redirected to the `...Large...` methods), so I'll remove it.